pages/login.js

```diff
@@ -0,0 +1,57 @@
+import Head from 'next/head';
+import styles from '../styles/Home.module.css';
+
+export default function Login() {
+
+  function onSubmit(event) {
+    event.preventDefault()
+
+    // Get form data and create JSON object
+    const formData = new FormData(event.target)
+    const formDataObj = {}
+    formData.forEach((value, key) => (formDataObj[key] = value))
+
+    let username = formDataObj["username"]
+    let password = formDataObj["password"]
+
+    // Check if user input username and password
+    if (username.length == 0 || password.length == 0) {
+      let error = document.getElementById('error-message');
+      error.innerText = "Please enter both a Username and Password."
+      return
+    }
```

Comment:
May be better to re-render the login component by using with [React states](https://legacy.reactjs.org/docs/hooks-state.html)? Check out the `useState `to update the `errors` variable in the `create-account.js`. What are your thoughts @jackfales?

Reply:
just fixed this. should be good now.